pages/common/ybacklight.md

```diff
@@ -0,0 +1,20 @@
+# ybacklight
+
+> A program for managing the screen backlight brightness, simpler alternative to `xbacklight`.
+> More information: <https://github.com/pixelcmtd/ybacklight>.
+
+- Print short current brightness "/" short maximal brightness:
+
+`ybacklight Sc/Sm`
+
+- Set the brightness to 420:
+
+`ybacklight s{{420}}`
+
+- Increase the brightness by 42 big steps:
```

Comment:
What "42 big steps" stands for? Is it something like 4200?

Reply:
The change makes sense, but it sounds really confusing. I'd just say `4200`.

Reply:
The problem here is that the factor is not always the same, it is designed to be changed in the `config.h`, if, for example, you have a display with a backlight that goes up to a really high value. That's why I put it that way instead of `4200`.

Reply:
How about this one:
```suggestion
- Increase the brightness by 42 big steps (4200 by default):
```